feature-libs/cart/quick-order/components/quick-order/form/quick-order-form.component.ts

```diff
@@ -24,7 +25,17 @@ export class QuickOrderFormComponent implements OnInit, OnDestroy {
   form: FormGroup;
   iconTypes = ICON_TYPE;
 
+  get isDisabled(): boolean {
+    return this._disabled;
+  }
+
+  @Input('isDisabled') set isDisabled(value: boolean) {
```

Comment:
for you information, 

since you aliased the input `@Input('isDisabled'), you can change the name to something like `setSomething` --> something meaningful, but you don't need to do this